### ISSUE TITLE
Is it possible to add new models of LiDAR that this software does not have(for examples livox), has anyone tried it?

### ISSUE THREAD

**Issue by zjj1105** [mgschwan/blensor#51]
Is it possible to add new models of LiDAR that this software does not have(for examples livox), has anyone tried it?

**Comment by yangfh2004** [comment]
You can try; this repo is somehow dead. How about you create a fork and keep maintaining it

**Comment by mgschwan** [comment]
Yes, adding additional models would be possible and should not be too hard, but I unfortunately do not have the resources for this in the forseeable future

**Comment by yccckid** [comment]
> Is it possible to add new models of LiDAR that this software does not have(for examples livox), has anyone tried it?

have you made it done?

**Comment by mgschwan** [comment]
I'm still pondering remaking blensor as a pure blender plugin but I don't know what the demand for that is

**Comment by yangfh2004** [comment]
> I'm still pondering remaking blensor as a pure blender plugin but I don't know what the demand for that is

We were using blensor for LIDAR algorithm research, we use it to simulate LIDAR data and feed it into our AI model.
